config/webpacker.yml

```diff
@@ -58,9 +58,9 @@ development:
   # Reference: https://webpack.js.org/configuration/dev-server/
   dev_server:
     https: false
-    host: localhost
+    host: <%= ENV["WEBPACKER_DEV_SERVER_HOST"] %>
     port: 3035
-    public: localhost:3035
+    public: <%= ENV["WEBPACKER_DEV_SERVER_URL"] %>
```

Comment:
We might want to use `fetch` and return `localhost` when the environment variable isn't available.